packages/drivekit_core/flutter_drivekit_core_android/android/src/main/kotlin/com/drivequant/drivekit/flutter/core/CoreApi.kt

```diff
@@ -12,112 +12,140 @@ import io.flutter.plugin.common.StandardMessageCodec
 import java.io.ByteArrayOutputStream
 import java.nio.ByteBuffer
 
-private fun wrapResult(result: Any?): List<Any?> {
-  return listOf(result)
-}
+private fun wrapResult(result: Any?): List<Any?> = listOf(result)
```

Comment:
It's a generated file, you should exclude it from your linter instead of manually editing it. Otherwise we'll need to do that each time we run `melos pigeon`

Reply:
If its never modified manually, it's better to ignore it, yes.
Done in new commits